hackathon_site/hardware/tests.py

```diff
@@ -121,28 +121,51 @@ def test_base_case_no_order_items(self):
     def test_some_items_none_returned(self):
         team = Team.objects.create()
         order = Order.objects.create(status="Picked Up", team=team)
```

Comment:
the order model has been changed, there is one more field called "requests". And I will merge that pr asap!

[https://github.com/ieeeuoft/hackathon-template/pull/332](url)